src/Microsoft.Azure.WebJobs.Logging.ApplicationInsights/ApplicationInsightsLoggerOptions.cs

```diff
@@ -36,7 +36,7 @@ public class ApplicationInsightsLoggerOptions : IOptionsFormatter
         /// Gets or sets the time to delay Quick Pulse (Live Metrics) initialization. Delaying this initialization
         /// can result in decreased application startup time. Default value is 15 seconds.
         /// </summary>
-        public TimeSpan QuickPulseInitializationDelay { get; set; } = TimeSpan.FromSeconds(15);
+        public TimeSpan LiveMetricsInitializationDelay { get; set; } = TimeSpan.FromSeconds(15);
```

Comment:
Thanks :-)

Reply:
Shoot -- I don't think we can change this as this has been released already. So -- can you create a second property called `LiveMetricsInitializationDelay` -- and mark the `QuickPulse` one as deprecated? We just won't ever use `QuickPulse` one internally.

Reply:
... it was never documented but it makes me nervous that someone could build locally against one version and deploy against another with a missing property.

Reply:
Ohh I didn't realize it had been released. Fair, I will make that update. Working through a couple of release issue. May get delayed until Monday

Reply:
Looks like there's also this setting `QuickPulseAuthenticationApiKey` that i need to update